### ISSUE TITLE
Brackets in atom:id can create an error in uri resolving

### ISSUE THREAD

**Issue by GVogeler** [icaruseu/mom-ca#734]
http://monasterium.net/mom/CZ-SOAT/VelkostatekOrlik/5%28a%29/charter did not work. After changing atom:id and filename to `tag:www.monasterium.net,2011:/charter/CZ-SOAT/VelkostatekOrlik/5_a_` it works (http://monasterium.net/mom/CZ-SOAT/VelkostatekOrlik/5_a_/charter). We could check if brackets are used in other atom:id .

**Comment by GVogeler** [comment]
```
xquery version "3.0";
declare namespace atom="http://www.w3.org/2005/Atom";
declare namespace cei="http://www.monasterium.net/NS/cei" ;
for $u in collection('/db/mom-data/metadata.charter.public/')//atom:id[contains(.,'%28')]
return $u
```
liefert 8856 Fälle.
Davon funktioniert nicht z.B. http://monasterium.net/mom/AT-DAL/Urkunden/Urkunden_PA%28Gr%C3%BCnau%29/charter

**Comment by GVogeler** [comment]
resolved by #732 ?
